### PR TITLE
Added new display Card

### PR DESCRIPTION
**Pull request by ZaidKhan2002**
Solved Issue: #60

### CODE REVIEW

File: Product_Display_Cards/Product_Card_04/code/style.css [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    padding: 0;
+    box-sizing: border-box;
+}
+
+body {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    min-height: 100vh;
+    background: #262626;
```
Comment (Abirpal202049) [inline Product_Display_Cards/Product_Card_04/code/style.css:12]: Don't target the Body Directly

**Review by Abirpal202049 — CHANGES_REQUESTED** [review]

File: Product_Display_Cards/Product_Card_04/code/style.css (hunk 1 above)
Comment (ZaidKhan2002) [reply Product_Display_Cards/Product_Card_04/code/style.css]: Made the changes!!

File: Product_Display_Cards/Product_Card_04/README.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+<h3> Tech Stack Used <img src = "https://media2.giphy.com/media/QssGEmpkyEOhBCb7e1/giphy.gif?cid=ecf05e47a0n3gi1bfqntqmob8g9aid1oyj2wr3ds3mg700bl&rid=giphy.gif" width = 32px> </h3>
+<p align ="left"> 
+<a href="https://www.w3.org/html/" target="_blank"> <img src="https://raw.githubusercontent.com/devicons/devicon/master/icons/html5/html5-original-wordmark.svg" alt="html5" width="40" height="40"/> </a> 
+<a href="https://www.w3schools.com/css/" target="_blank"> <img src="https://raw.githubusercontent.com/devicons/devicon/master/icons/css3/css3-original-wordmark.svg" alt="css3" width="40" height="40"/> </a>
+</p>
+
+### Preview
+
+![](https://github.com/Dezenix/frontend-html-css-js/blob/main/Product_Display_Cards/Product_Card_04/assets/Product_Display_Card.gif)
```
Comment (thepranaygupta) [inline Product_Display_Cards/Product_Card_04/README.md:11]: embed the preview gif directly by drag and drop, also try to optimize it under 10MB

**Review by thepranaygupta — CHANGES_REQUESTED** [review]

File: Product_Display_Cards/Product_Card_04/README.md (hunk 2 above)
Comment (ZaidKhan2002) [reply Product_Display_Cards/Product_Card_04/README.md]: .mov is not supported so I converted to .gif online and compressed it under 10MB
Product-Card-Preview

**Review by Abirpal202049 — APPROVED** [review]

**Review by thepranaygupta — APPROVED** [review]

**Comment by thepranaygupta** [comment]
This PR can be merged now 👍

**Pull request merged**